src/lm-profile-status.html

```diff
@@ -1,48 +0,0 @@
-<link rel="import" href="../bower_components/polymer/polymer-element.html">
-<link rel="import" href="../bower_components/iron-ajax/iron-ajax.html">
-
-
-<dom-module id="lm-profile-status">
-  <template>
-    <iron-ajax
-               auto
-               url="data/profile-status.json"
-               handle-as="json"
-               last-response="{{ajaxResponse}}"
-               >
-    </iron-ajax>
-    <span>{{ajaxResponse.responseMessage}}</span>
-  </template>
-  <script>
-    class ProfileStatus extends Polymer.Element {
-      static get is() { return 'lm-profile-status'; }
-      static get properties() {
-        return {
-          prop1: {
```

Comment:
Remove prop1 and provide meaningful names to the variable

Reply:
Ok I ll take care